src/crypto/rand/rand_stm32.go

```diff
@@ -0,0 +1,40 @@
+//go:build stm32wle5
+// +build stm32wle5
+
+package rand
+
+import (
+	"machine"
+)
+
+func init() {
+
+	Reader = &reader{}
+	machine.InitRNG()
+}
+
+type reader struct {
+}
+
+func (r *reader) Read(b []byte) (n int, err error) {
+	if len(b) == 0 {
+		return
+	}
+
+	var randomByte uint32
+	for i := range b {
+		// Call rand_s every four bytes because it's a C int (always 32-bit in
+		// Windows).
```

Comment:
This looks like a copied comment. It is not relevant here because this is not Windows.